src/components/EditImageModal/index.jsx

```diff
@@ -342,29 +416,78 @@ export default class EditImageModal extends React.Component {
   getImage = () => (
     <img
       alt=""
+      className={classNames(styles['image-preview-image'], { invisible: !this.state.isImageLoaded })}
       src={this.getImageAssetSource()}
       onLoad={this.onImageLoad}
       onError={this.onImageError}
       ref={(input) => { this.imgRef = input; }}
     />
   );
 
+  /*
+    image is conditionally displayed so that onImageError is not called on initial
+    modal open, when image source is empty string
+  */
   getImagePreviewPlaceholder = () => (
-    <div className="image-preview"><WrappedMessage message={messages.editImageModalImagePreviewText} /></div>
+    <div className={styles['image-preview-placeholder']}>
+      <WrappedMessage message={messages.editImageModalImagePreviewText} >
+        {displayText =>
+          (<span className={classNames({ invisible: this.state.isImageLoaded })}>
+            {displayText}
+          </span>)}
+      </WrappedMessage>
+      { this.state.imageSource && this.getImage() }
+    </div>
   );
 
-
   getImagePreview = () => (
-    <React.Fragment>
-      <div className="row">
-        <span><WrappedMessage message={messages.editImageModalImagePreviewText} /></span>
+    // image preview is decorative
+    <div aria-hidden>
+      <WrappedMessage message={messages.editImageModalImagePreviewText} />
+      <div className={styles['image-preview-container']}>
+        {this.getImagePreviewPlaceholder()}
       </div>
-      {this.state.imageSource ? this.getImage() : this.getImagePreviewPlaceholder()}
-    </React.Fragment>
+    </div>
+  );
+
+  getStatusAlertDialog = () => (
+    <div>
+      <WrappedMessage
+        message={messages.editImageModalFormErrorMissingFields}
+        tagName="div"
+      />
+      <br />
```

Comment:
We should use styling here instead of the `<br />`. 

[The br element may only be used for line breaks that are actually part of the content.](https://www.w3.org/TR/2014/REC-html5-20141028/text-level-semantics.html#the-br-element) We want this to be a visual indicator of space between the message and the list, so it would be better practice to add a class to adjust the margin/padding instead of using this tag.

Reply:
That sounds good. Thank you for correcting that. I copied it from the [accessibility page|https://github.com/edx/studio-frontend/blob/25bd9cefb27514511be7c23153696baa767f3b9e/src/components/AccessibilityPolicyForm/index.jsx#L94], so we should be sure to make that change there too.